code/modules/clothing/shoes/miscellaneous.dm

```diff
@@ -544,3 +544,43 @@
 /obj/item/clothing/shoes/gunboots/Initialize()
 	. = ..()
 	AddComponent(/datum/component/projectile_shooter, projectile_type = projectile_type, shot_prob = shot_prob, signal_or_sig_list = list(COMSIG_SHOES_STEP_ACTION, COMSIG_HUMAN_MELEE_UNARMED_ATTACK))
+
+//Nemesis Soutions boots
+
+/obj/item/clothing/shoes/bhop/nemesis
+	name = "combat jackboots"
+	desc = "A pair of modified jackboots from Nemesis Solutions, equipped with a propulsion system."
+	icon_state = "nemesis"
+
+	jumpdistance = 4
+	recharging_rate = 2 SECONDS
+	actions_types = list(/datum/action/item_action/bhop/nemesis)
+
+/obj/item/clothing/shoes/bhop/nemesis/ui_action_click(mob/user, action)
```

Comment:
you should probably rethink this proc, its a subtype of bhop shoes but it uses copypasted bhop shoes proc with some changes instead of calling parent